health/notifications/alarm-notify.sh.in

```diff
@@ -1901,6 +1901,9 @@ send_hangouts() {
 		if [ -z "${HANGOUTS_WEBHOOK_URI[$room]}" ] ; then
 			info "Can't send Hangouts notification for: ${host} ${chart}.${name} to room ${room}. HANGOUTS_WEBHOOK_URI[$room] not defined"
 		else
+			if [ -n "${HANGOUTS_WEBHOOK_THREAD[$room]}" ]; then
```

Comment:
do we need to `declare HANGOUTS_WEBHOOK_THREAD`?

like https://github.com/thiagoftsm/netdata/blob/0b979a2fe45e4434b260141fa88c4729556984ce/health/notifications/alarm-notify.sh.in#L375-L376

Reply:
Thank you @ilyam8 !